modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/getWorkflowDefinition.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```

Comment:
Hey @larissacribeiro, could you use the current year here?

Reply:
yess, my bad, thanks!